### ISSUE TITLE
Redesign and rewrite Phenogrid component

### ISSUE THREAD

**Issue by putmantime** [monarch-initiative/monarch-app#107]
The Phenogrid component, viewable on the current Monarch site under tools -> phenotype profile search, is an external package, and is sort of a microcosm of the Monarch web app itself. It needs to be redesigned from a UI perspective, and its code needs to be modernized, refactored, and fixed.

This will likely need to be split up into several sub issues. This can also likely be done in parallel or at a later time, as it is somewhat sandboxed from the rest of the application. However, it may not be *completely* sandboxed, as how we redesign this component may make some of the walkthrough steps in the 2.0 "phenotype profile search wizard" unnecessary (we may end up merging some of that functionality right into the component itself).

**Comment by vincerubinetti** [comment]
We don't currently have a tag that's like "ultra-enhancement", or "very long term nice-to-have", so I'm going to close this as won't fix (any time in the near future). It can be reopened if it's ever back on the roadmap.

Moreover, this is really outside the scope of this repo, and should be an issue on the phenogrid repo instead of this repo.
